src/Encamina.Enmarcha.Bot.Skills.QuestionAnswering/QuestionAnsweringDialog.cs

```diff
@@ -59,9 +59,9 @@ public QuestionAnsweringDialog(string id, QuestionAnsweringDialogServices questi
     public override string Name => string.IsNullOrWhiteSpace(configurationOptions.DialogName) ? base.Name : configurationOptions.DialogName;
 
     /// <inheritdoc/>
-    public override async Task<DialogTurnResult> BeginDialogAsync(DialogContext dc, object options = null, CancellationToken cancellationToken = default)
+    public override async Task<DialogTurnResult> BeginDialogAsync(DialogContext dc, object? options = null, CancellationToken cancellationToken = default)
```

Comment:
Setting `object?` is somehow breaking the public interface of the `Microsoft.Bot.Builder.Dialogs.Dialog` type.
For now, revert this change and create a new specific issue to address this part of the code.